console.go

```diff
@@ -1,3 +1,19 @@
+/*
+   Copyright The containerd Authors.
```

Comment:
i actually didnt notice for the rest of the merged PRs, but shouldnt this have date. The boilerplate is `Copyright {yyyy} {name of copyright owner}`

Reply:
No, date is actually not required and recommended to not have it so that the files don't have to be updated each year/